### PR TITLE
feat(search): focus search input on forward slash

### PR DESCRIPTION
**Pull request by dzucconi**
This is just one of those things I keep trying to do and being vaguely frustrated when it doesn't work. I'm a little unclear on the origin of this keyboard shortcut but it is pervasive at this point.

### CODE REVIEW

File: src/Components/Search/NewSearch/SearchBarInput.tsx [hunk 1]
```diff
@@ -177,8 +191,33 @@ const SearchBarInput: FC<SearchBarInputProps> = ({
...
+    const handleKeyUp = ({ target, key }: KeyboardEvent) => {
+      if (!ref.current || key !== "/") return
+
+      const tag =
+        (target && (target as HTMLElement).tagName.toLowerCase()) || ""
+
+      // Ignore if an input has focus
+      if (["input", "textarea", "select"].includes(tag)) {
+        return
+      }
```
Comment (anandaroop) [inline src/Components/Search/NewSearch/SearchBarInput.tsx:207]: **question (non-blocking):** Does a `select` fire keyup events?

**Review by anandaroop — APPROVED** [review]
Nice!

>  I'm a little unclear on the origin of this keyboard shortcut 

Dare I say `vi`? 

Feels like a geeky shortcut to include in a general-audience product, but Gmail has had it forever and I've often wished we did too 👍🏽

File: src/Components/Search/NewSearch/SearchBarInput.tsx (hunk 1 above)
Comment (dzucconi) [reply src/Components/Search/NewSearch/SearchBarInput.tsx:207]: Yep, you can tab into one then start typing which will select corresponding options.

**Review by dblandin — APPROVED** [review]
Nice 💯

**Review by nickskalkin — DISMISSED** [review]

**Comment by damassi** [comment]
Can we merge this @dzucconi ?

**Pull request merged**